app/navigation/Navigator.scala

```diff
@@ -252,5 +263,14 @@ class Navigator @Inject()() {
     }
   }
 
+  private def secondaryWarehouseDetailsPageNavigation(userAnswers: UserAnswers) = {
```

Comment:
have all of these flows been tested

Reply:
![image](https://user-images.githubusercontent.com/75027964/233585109-d45ed158-47a1-4d79-b3a8-e20fcbdfa6de.png)

Reply:
added nav test for bot true and false, true needs to be changed when alf is available.